tests/mock/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+use fuel_pbft::*;
+
+use curve25519_dalek::constants;
```

Comment:
Think we could eventually simulate this with `sepk256` eventually? For now this curve is totally fine and even ideal as we mock up everything.

Reply:
Yes, the choice of curve here is arbitrary